package.json

```diff
@@ -42,8 +42,10 @@
   "devDependencies": {
     "chai": "^3.0.0",
     "debug": "^2.2.0",
-    "eslint": "^4.1.1",
+    "eslint": "^4.16.0",
     "eslint-config-homeoffice": "^2.1.2",
+    "eslint-plugin-filenames": "^1.2.0",
+    "eslint-plugin-mocha": "^4.11.0",
```

Comment:
Orly https://github.com/UKHomeOffice/rotm/blob/master/package.json#L39-L51

Reply:
Nah, you just need to update `eslint-config-homeoffice`

Reply:
ahh ok